static/bidder-info/frvradn.yaml

```diff
@@ -0,0 +1,18 @@
+endpoint: "https://fran.frvr.com/api/v1/openrtb"
+maintainer:
+  email: "engineering@frvr.com"
```

Comment:
@franfrvr verification is sent at `engineering@frvr.com`.  we kindly request to respond with "received" message to confirm the email address.

Reply:
@franfrvr email has bounced back. Please update to a working email address.

Reply:
@gargcreation1992 @onkarvhanumante 
apologies, it has been updated.

Reply:
@franfrvr verification mail sent on `frvr.limited@frvr.com` has bounce back. Please check the updated email address

Reply:
@franfrvr  email is sent on `info@frvr.com`. Please respond back with "received" message to confirm the email address.

Reply:
@onkarvhanumante thanks, responded not long ago